### DISCUSSION TITLE
Tackling class imbalance in PyTorch Geometric

### DISCUSSION THREAD

**Discussion by JopVerbeek** [pyg-team/pytorch_geometric#2468] [Q&A]
Hi, 

I am currently building a graph classification network using PyTorch Geometric. The dataset I am using unfortunately has large class imbalances. I was wondering if there is a straightforward method to tackle this, like for example in regular PyTorch I can implement WeightedRandomSampler() in the DataLoader itself. Is this maybe also possible in the PyTorch Geometric DataLoader or do I need alter the DataLoader class in order to be able to use, for example, WeightedRandomSampler?

Thanks in advance!

**Comment by rusty1s** [comment]
Yes, you can also do that in PyG, too. We make use of the PyTorch `DataLoader` under the hood, hence, all arguments of the PyTorch `Dataloader` can be passed as well (expect for `collate_fn`).
